src/main/java/uk/gov/justice/hmpps/prison/api/resource/IncidentsResource.java

```diff
@@ -33,11 +34,11 @@ public class IncidentsResource {
             @ApiResponse(responseCode = "400", description = "Invalid request.", content = {@Content(mediaType = "application/json", schema = @Schema(implementation = ErrorResponse.class))}),
             @ApiResponse(responseCode = "404", description = "Requested resource not found.", content = {@Content(mediaType = "application/json", schema = @Schema(implementation = ErrorResponse.class))}),
             @ApiResponse(responseCode = "500", description = "Unrecoverable error occurred whilst processing request.", content = {@Content(mediaType = "application/json", schema = @Schema(implementation = ErrorResponse.class))})})
-    @Operation(summary = "Return an Incident for a given incident ID", description = "System access only")
+    @Operation(summary = "Return an Incident for a given incident ID", description = "Requires the VIEW_PRISONER_DATA role.")
     @GetMapping("/{incidentId}")
+    @PreAuthorize("hasAnyRole('SYSTEM_USER', 'VIEW_PRISONER_DATA')")
```

Comment:
Added VIEW_PRISONER_DATA  which the client already has

Reply:
do we need to add SYSTEM_USER ? (FYI @karenmillermoj )

Reply:
Actually, where is this API used? Is it Prisoner Profile or in some incident reporting ? Might be a opportunity to add a different role - this feels very incident specific rather something that should be exposed to some one that just needs core prisoner data. VIEW_INCIDENTS might make more sense, but not sure if this is getting into @karenmillermoj  territory

Reply:
I'm happy to tidy this up as part of my work @steverendell

Reply:
OK. The only client is the risk-profiler which currently has SYSTEM_USER and VIEW_PRISONER_DATA. So i could protect this with `hasAnyRole('SYSTEM_USER', 'VIEW_INCIDENTS')`. There are other incident endpoints which should be addressed too though, I could look at them next.

Reply:
Ill do this for the 2 risk-profiler endpoints and remove the others which are not called